lib/mongo/monitoring/event/cmap.rb

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2016-2019  MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  class Monitoring
+    module Event
+
+      # Base class for CMAP events.
+      #
+      # @since 2.8.0
+      class Cmap < Mongo::Event::Base
```

Comment:
Please make Cmap a module and Cmap::Base the base class.

Reply:
Okay

Reply:
Done